components/build-service/base/build-pipeline-config/build-pipeline-config.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: build-pipeline-config
+  namespace: build-service
+data:
+config.yaml: |
+  pipelines:
+  - name: fbc-builder
+    bundle: quay.io/redhat-appstudio-tekton-catalog/pipeline-fbc-builder:032a8745d43a942a247f365fc890b06023ccd67d
+  - name: docker-builder
+    bundle: quay.io/redhat-appstudio-tekton-catalog/pipeline-docker-build:032a8745d43a942a247f365fc890b06023ccd67d
```

Comment:
When can we get away from using the term "docker"? 😅

Reply:
suggest something else